services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/mapper/annotations/DynamoDbBean.java

```diff
@@ -46,9 +46,11 @@
  * </ul>
  *
  * Example using attribute converter providers with one custom provider and the default provider:
+ * <pre>
  * {@code
  * (converterProviders = {CustomAttributeConverter.class, DefaultAttributeConverterProvider.class});
  * }
```

Comment:
Does @code work when the content has a } in it?

Reply:
Only if all the brackets are matched, if not, there will be an error.